tests/common/test_behav.py

```diff
@@ -101,8 +101,11 @@ def test_pos_interval_no_transaction(verbose_context, common, mini_restr):
         common.PositionIntervalMap()._no_transaction_make(mini_restr)
     after = common.PositionIntervalMap().fetch()
     assert (
-        len(after) == len(before) + 2
+        len(after) == len(before) + 3
     ), "PositionIntervalMap no_transaction had unexpected effect"
+    assert (
+        "" in after["position_interval_name"]
```

Comment:
I think this will always evaluate to True:
- `"" in ""`
- `"" in "aa"`
- `"" in "aa "`

Can we adjust this to reflect the contents of the null insert? `after["position_interval_name"] is ""` ?

Reply:
`after` comes from fetching the whole table, so `after["position_interval_name"]` should be a list of strings rather than a string.  This is to test that one of them is the null insert

Reply:
ah, my bad. thanks!